xml/docu_styleguide.outline.xml

```diff
@@ -307,6 +296,22 @@
      </para>
     </listitem>
    </varlistentry>
+    <varlistentry>
+     <term>Contributors</term>
+     <listitem>
+      <para>
+       Writing documentation is a collaborative effort. To also give credit to
+       external contributors, add an appendix to the guides, which points to the
+       <guimenu>Contributors</guimenu> page for the respective GitHub
+       repository. For an example, see <xref linkend="sec-contributors"/>.
+      </para>
+      <para>
+       For &suse; Best Practices and other articles and small documents whose
```

Comment:
```suggestion
       For &suse; Best Practices, other articles, and small documents whose
```

Reply:
As I interpret it, these are supposed to be two listings, nested within each other: `SBP + (other articles + small docs)`. I would suggest this, which is also not strictly equivalent but better I think:

```suggestion
       For articles and small documents (such as &suse; Best Practices) whose
```

Reply:
:+1: to @sknorr's suggestion.